app.json

```diff
@@ -0,0 +1,106 @@
+{
+  "name": "Indielogin",
+  "description": "Deploy Indielogin to heroku",
+  "repository": "https://github.com/aaronpk/indielogin.com",
+  "logo": "https://github.com/aaronpk/indielogin.com/raw/master/public/icons/android-chrome-512x512.png",
+  "website": "https://indielogin.com",
+  "keywords": ["indielogin", "nginx", "php", "indieauth"],
+  "addons": [
+    "heroku-redis",
+    {
+      "plan": "cleardb:ignite"
+    }
+  ],
+  "env": {
+    "BASE_URL": {
+      "description": "The base URL of your application including port",
+      "value": "",
+      "required": false
+    },
+    "APP_NAME": {
+      "description": "The application name",
+      "value": "IndieLogin.com",
+      "required": false
+    },
+    "DB_HOST": {
```

Comment:
Presumably on Heroku someone will be deploying this with a database provided by Heroku so only the `DATABASE_URL` option should be here? Or even leave that out since it will be set automatically?

Reply:
This gives a person choice to not use Heroku. Possibly they have a database they don't have or don't know how to make a URL for. Happy to remove if needed

Reply:
I guess then just add some clarifying text here to note that if they are using a database add-on from Heroku then they don't need to set these four values.

Reply:
I'll remove as with meetable